cmd/influxd/backup/backup.go

```diff
@@ -541,8 +542,13 @@ func (cmd *Command) download(req *snapshotter.Request, path string) error {
 		}(); err == nil {
 			break
 		} else if err != nil {
-			cmd.StderrLogger.Printf("Download shard %v failed %s.  Retrying (%d)...\n", req.ShardID, err, i)
-			time.Sleep(time.Second)
+			min := 2 * time.Second
```

Comment:
Can you move this out of the loop. Reinitialising it ten times is bothering me 😛